lib/drivex.js

```diff
@@ -115,6 +135,27 @@ DriveX.prototype = {
       throw new Error(msg || '[drivex.waitForElementVisible] Element not visible');
     });
   },
+    /**
+   * Wait for timeout milliseconds for the WebElement to be NOT visible
+   * @param locator {LocatorJSON}
+   * @param timeout {Number}
+   * @param msg {String} optional message for any error messages
+   * @returns {Promise} resolves to true or throw error
+   */
+  waitForElementNotVisible: function (locator, timeout, msg) {
+    var self = this;
+    log('waitForElementNotVisible', locator);
+    return this.driver.wait(function() {
+      return self.notVisible(locator);
```

Comment:
I see that having the `notVisible` method makes this method a little bit simpler. But I'm not convinced it makes this method enough simpler to necessitate the additional method and testing around it.